tools/single_use/Remove_Data_entry_completion_status_instr_column.php

```diff
@@ -44,8 +44,9 @@
 );
 
 // Get instrument names
-$instruments = [];
-foreach (\Utility::getAllInstruments() as $testname => $fullName) {
+$instruments    = [];
+$instrumentList = \NDB_BVL_Instrument::getInstrumentNamesList($loris);
```

Comment:
this should properly be replaced with getInstrumentsList since one of the first things it does is instantiate the instrument which is already done upstream in https://github.com/aces/Loris/blob/main/php/libraries/NDB_BVL_Instrument.class.inc#L2986

Reply:
the deprecation notice said to change it to getInstrumentNamesList. whats the difference between the two? should the deprecation notice be modified?

Reply:
ohh i see, getInstrumentsList returns the instrument object already 👍